overrides/ubuntu/22.04/overrides.yml

```diff
@@ -87,4 +87,27 @@
 # this is only making the vizkit dependency optional
 #- slam/orogen/slam3d:
 #  github: pierrewillenbrockdfki/ric-orogen-slam3d
-#  branch: optional-vizkit
\ No newline at end of file
+#  branch: optional-vizkit
+
+# https://github.com/rock-cpp/orogen_model_exporter/pull/12
+- tools/orogen_model_exporter:
+  branch: fix-boost-DEPS
+
+# https://github.com/rock-core/tools-graph_analysis/pull/8
+- tools/graph_analysis:
+  branch: fix-boost-DEPS
+
+# https://github.com/rock-slam/slam-orogen-pcl/pull/7
+- slam/orogen/pcl:
+  github: pierrewillenbrockdfki/slam-orogen-pcl
+  branch: pcl12-support
+
+# https://github.com/rock-multiagent/multiagent-fipa_services/pull/1
+- multiagent/fipa_services:
+  github: chhtz/multiagent-fipa_services
+  branch: fix/replace_boost_with_std
```

Comment:
These two are merged now (also a follow-up-fix for the latter: https://github.com/rock-multiagent/multiagent-fipa_services/pull/3)